src/addons/addonmessage.h

```diff
@@ -19,7 +19,7 @@ class AddonMessage final : public Addon {
   ADDON_PROPERTY(subtitle, m_subtitle, retranslationCompleted)
 
   Q_PROPERTY(Composer* composer READ composer CONSTANT)
-  Q_PROPERTY(bool isRead MEMBER m_isRead NOTIFY isReadChanged)
```

Comment:
Note: I removed the NOTIFY here because it was not being used anywhere.

Reply:
TODO: Change this back, this is totally used [https://github.com/mozilla-mobile/mozilla-vpn-client/blob/3e62d1245fa553311763b26d[…]ed9b5d2ad1836dba/src/ui/screens/messaging/ViewMessagesInbox.qml](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/3e62d1245fa553311763b26ded9b5d2ad1836dba/src/ui/screens/messaging/ViewMessagesInbox.qml#L290)... Thanks @MattLichtenstein for confirming!